manifests/client/packages.pp

```diff
@@ -8,10 +8,4 @@
   package { 'ipa-client':
     ensure => $simp_ipa::client::install::ipa_client_ensure,
   }
```

Comment:
simp_ipa::client::install::ipa_client_ensure is only for EL6 and should be marked as deprecated.  I will fix